web-ui/src/main/resources/catalog/templates/editor/editor.html

```diff
@@ -1,7 +1,10 @@
 <div data-ng-hide="metadataFound">
-  <div class="alert alert-warning" 
+  <div data-ng-show="!metadataLocked" class="alert alert-warning" 
     data-translate="" 
     data-translate-values="{ id: '{{metadataNotFoundId}}'}">metadataNotFound</div>
+    
+  <div data-ng-show="metadataLocked" class="alert alert-warning" 
```

Comment:
I can't see any messages if a record is locked. if the metadataFound == true line 1, this status message can't be displayed in any case. Another option would be to simply display the error message return by the edit service ?

![image](https://cloud.githubusercontent.com/assets/1701393/15421970/40fe2970-1e76-11e6-895b-dfceb8ad2173.png)

... and even start returning i18n exceptions (https://github.com/geonetwork/core-geonetwork/blob/develop/services/src/main/java/org/fao/geonet/api/user/PasswordApi.java#L105) to not have to handle that client side.